lib/webtrit_adapter_client.ex

```diff
@@ -230,6 +230,28 @@ defmodule WebtritAdapterClient do
     request(client, options)
   end
 
+  @spec invoke_custom_public_method(Tesla.Client.t(), String.t(), String.t(), map()) :: result()
+  def invoke_custom_public_method(client, method_name, extra_params, data) do
```

Comment:
Why `invoke`? To be in consist with other method names I propose to use actual HTTP method - `post`.

Reply:
After further discussion, we decided that the `invoke` prefix is more suitable and logical here.